packages/react/src/components/Forms/DatePicker/index.scss

```diff
@@ -30,6 +30,12 @@
       display: flex;
       align-items: center;
     }
+
+    &__toggle {
+      .material-icons {
```

Comment:
Is it material relative to the material icon lib ?
I think we should use a name of class that's not related to any library name.